Pages/ChildSaveBank.js

```diff
@@ -0,0 +1,58 @@
+import { StatusBar } from 'expo-status-bar';
+import React from 'react';
+import { NavigationContainer } from '@react-navigation/native';
+
+import { Button, StyleSheet, Text, View } from 'react-native';
+
+
+function ChildSaveBank({ navigation }) {
+    return (
+    <View style={styles.container}>
+         <Button 
+            style={styles.BackButton}
+            title="<--"
+            onPress={() => navigation.goBack()}
+        />
+        <Text>Save Bank:</Text>
+      
+        <Button 
+            style={styles.NewSaveTransaction}
+            title="New Transaction"
+            onPress={() => navigation.navigate('EnterSaveTransaction')}
+        />
+        <Text>Past Transactions:</Text>
+        <Text>Tooth fairy money +$4.00</Text>
+        <Text>Grandma gave me money for being nice +$2.00</Text>
+        <Text>Mom took away money for cursing -$5.00</Text>
+    
+        </View>
+    )
+}
+
+const styles = StyleSheet.create({
+    container: {
+        flex: 1,
+        backgroundColor: '#fff',
+        alignItems: 'center',
+        justifyContent: 'center',
+    },
+    BackButton: {
```

Comment:
consistent naming convention on capitalization.

Reply:
Also, this looks like the same code as mentioned above. Try to have a theme that is used in multiple places